chart/mits/templates/cfcred.yaml

```diff
@@ -0,0 +1,28 @@
+{{- /*
+   Copyright 2020 SUSE
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/ -}}
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ printf "%s-cf-cred" .Release.Name | quote }}
```

Comment:
Just a good practice to avoid possible collisions when reusing an existing namespace. Let's say we want to run this on the same namespace as KubeCF, then for some reason, KubeCF creates a secret with the same name. BOOM!